lib/reserve.rb

```diff
@@ -0,0 +1,30 @@
+require "date"
+require_relative "room"
+require "awesome_print"
+
+class Reservation
```

Comment:
Try to keep your file/class names in parity, otherwise it's hard to know which files to open.